.github/workflows/build.yml

```diff
@@ -6,22 +6,23 @@ on:
   push:
     branches:
       - main
+    tags:
+      - 'v*'
```

Comment:
Why do we want to run on tags?

Reply:
My initial thought was that if for any reason we have to manually force push a release tag, we should run basic workflows build, test and lint.
The current `make release` target does not cover all of them.

Reply:
Gotcha. makes sense